src/main/webapp/scripts/ou-script.js

```diff
@@ -1,26 +1,105 @@
+// Notes to self
+// IF we have no orgunits in admin console, response.json()['organizationUnits'] is undefined
+// blockInheritance can be undefined for some orgunits in our response, means it was never set
+// we can use the API to return all org units STARTING at a certain org unit (makes layers up / layers down code easy)
+// because not saving on API calls (one to get all OUs), just have limit layers make those 3 layers expanded rather than ONLY showing those 3 layers OR reword limit depth to limit display of or something
+
+// prevent visualization from being potentially dragged off the screen
+// determine size for svg based on size of visualization somehow?
+// layers and search work independently now (aka you have to hit Render) to get layer limiting, won't just do it on search. might want to change that
+
+
+
+// clean up path that appears in delete modal
+
 // Show refresh button and overlay
 var isLoading;
 
 // Org Units data in JSON forms
 var orgUnits;
 var parentChildOUs;
 
-// Search and Display criteria
+// Search criteria
+var searchName;
+
+
+var hereAreTheSearchedOUs;
```

Comment:
this variable should be renamed or removed, it seems like it is usually the OUs that match the search parameters, 
if that is the case, then removing it and replacing it with a method that returns all ous matching the search paramters would be better

otherwise something like ousMatchingSearchCriteria would be better

Reply:
Yup, so the reasoning for this was that the limit layers code relies on having a global variable storing the OUs (in this case if a user is using both search and limit layers, they need a copy of the search filtered OUs to then layer limit it). but yes, definitely intended on renaming this variable in my next commit - I just gave it a very easily searchable name for the timebeing so it would be easy to replace across all files haha.